tracer/src/Datadog.Trace/Configuration/ImmutableExporterSettings.cs

```diff
@@ -94,6 +94,11 @@ public ImmutableExporterSettings(ExporterSettings settings)
         /// </summary>
         public int PartialFlushMinSpans { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether metrics are computed on the tracer side
+        /// </summary>
+        public bool TracerStatsEnabled { get; }
```

Comment:
I don't think this should be on both `ImmutableExporterSettings` and `ImmutableTracerSettings`? I think it should either be on 

- `ExporterSettings` & `ImmutableExporterSettings`
- `TracerSettings` & `ImmutableTracerSettings`

Reply:
Right, my bad. I wasn't even assigning the property in ImmutableExporterSettings 😅

Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2591/commits/d81db7e2d205073e9bd5284b307bf6326fbd9f24